dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleAPI.java

```diff
@@ -168,4 +169,16 @@
 	 */
 	public void deleteAssetFromBundle(String assetId, String bundleId) throws DotDataException;
 
+	/**
+	 * This takes a Bundle, generates the folder/file structure and returns the resulting directory
+	 * @param bundleId
+	 * @return
+	 */
+    public File generateBundleDirectory(Bundle bundle);
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'generateBundleDirectory': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6239861)